addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -732,13 +732,20 @@ export class CalendarModel extends Model {
             type: "dynamic",
             recordId: null,
             value,
-            label: formatter(rawValue, { field }) || _t("Undefined"),
+            label: formatter(rawValue, { field }) || this.defaultFilterLabel,
             active: previousFilter ? previousFilter.active : true,
             canRemove: false,
             colorIndex,
             hasAvatar: !!value,
         };
     }
+    /**
+     * @protected
+     * @returns {string}
+     */
+    get defaultFilterLabel() {
+        return _t("Undefined");
+    }
```

Comment:
You did not move the getter. 😬 

Can you move it below `quickCreateFormViewId`? Thanks in advance!

Reply:
Done @xavierbol